libs/client-api/src/http_publish.rs

```diff
@@ -113,6 +65,18 @@ impl Client {
 
 // Guest API (no login required)
 impl Client {
+  pub async fn get_published_collab_info(
+    &self,
+    view_id: &uuid::Uuid,
```

Comment:
Is the workspace_id not required in this API?

Reply:
not required